CMakeLists.txt

```diff
@@ -181,7 +181,7 @@ if(BUILD_TESTING)
     osrf_testing_tools_cpp::memory_tools LIBRARY_PRELOAD_ENVIRONMENT_IS_AVAILABLE)
 
   ament_add_gtest(test_logging test/test_logging.cpp)
-  target_link_libraries(test_logging ${PROJECT_NAME} osrf_testing_tools_cpp::memory_tools)
+  target_link_libraries(test_logging ${PROJECT_NAME} mimick osrf_testing_tools_cpp::memory_tools)
```

Comment:
How many libraries need to be added before these calls should be separated into multiple lines?

Reply:
Hmm, I don't think we have a policy -- `lint_cmake` is not very pedantic. But I don't mind wrapping.